src/wp-includes/link-template.php

```diff
@@ -1597,25 +1597,39 @@ function get_delete_post_link( $post = 0, $deprecated = '', $force_delete = fals
  * @since 2.3.0
  *
  * @param int|WP_Comment $comment_id Optional. Comment ID or WP_Comment object.
- * @return string|void The edit comment link URL for the given comment.
+ * @param string         $context    Optional. How to output the '&' character. Default '&amp;'.
```

Comment:
I think the way you had it before was still more helpful, my previous comment https://github.com/WordPress/wordpress-develop/pull/7071#discussion_r1699129591 was only referring to the duplicate value as problematic.

I think this would work:
```suggestion
 * @param string         $context    Optional. Context in which the URL should be used. Either 'display',
 *                                   to include HTML entities, or 'url'. Default 'display'.
```